docs/2.9.0/docs/canton/usermanual/installation.rst

```diff
@@ -249,64 +249,64 @@ Canton distinguishes static from dynamic configuration.
 * :ref:`Static configuration <static_configuration>` are items which are not supposed to change and are therefore captured in the configuration file.
   An example is to which port to bind to.
 
-* Dynamic configuration are items such as Daml archives (DARs), domain connections or parties. All such changes are effected
+* Dynamic configuration are items such as Daml archives (DARs), sync domain connections, or parties. All such changes are effected
   through :ref:`console commands <canton_console>` (or the :ref:`administration APIs <administration_apis>`).
 
-If you don't know how to connect to domains, onboard parties or provision Daml code, please read the
+If you don't know how to connect to sync domains, onboard parties, or provision Daml code, please read the
 :ref:`getting started guide <canton-getting-started>`.
 
-Setting up a Synchronization Domain
------------------------------------
+Setting up a Sync Domain
```

Comment:
In headers, keep using "Synchronization Domain" as a reminder of what "sync" means